priv/repo/migrations/20241021182426_add_invalidated_by_user_id.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Oli.Repo.Migrations.AddInvalidatedByUserId do
+  use Ecto.Migration
+
+  def change do
+    alter table(:payments) do
+      add :invalidated_by_user_id, :integer, null: true
```

Comment:
Why are we setting this as an integer and not as a reference to a user?

Reply:
To be consistent with how we implemented the `bypassed_by_user_id` field.

Reply:
It makes sense.